src/contrib/mlir/CMakeLists.txt

```diff
@@ -0,0 +1,118 @@
+# ******************************************************************************
+# Copyright 2019 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ******************************************************************************
+
+set(SRC
+    dialect/dialect.cpp
+    dialect/type.cpp
+    dialect/ops.cpp
+    compiler.cpp
+    lowerer.cpp
+    memory_manager.cpp
+    pass/mlir_subgraph_extraction.cpp
+    pass/mlir_subgraph_extraction.hpp
+)
+if (NGRAPH_MLIR_ENABLE)
+
+    add_library(mlir_backend SHARED ${SRC})
+
+    message(STATUS "LLVM Directory: ${LLVM_DIR}")
+    # Link LLVM and MLIR
+    find_package(LLVM REQUIRED CONFIG)
+    
+    set(MLIR_LLVM_INCLUDEPATH ${LLVM_INCLUDE_DIRS})
+
+    message(STATUS "Found LLVM ${LLVM_PACKAGE_VERSION}")
+    message(STATUS "Using LLVMConfig.cmake in: ${LLVM_DIR}")
+    message(STATUS "LLVM RTTI is ${LLVM_ENABLE_RTTI}")
+    add_definitions(${LLVM_DEFINITIONS})
+   
+    target_include_directories(mlir_backend PRIVATE ${LLVM_INCLUDE_DIRS})
+    
+    message(STATUS "MLIR Headers at : ${MLIR_INCLUDE_PATHS}")
+    message(STATUS "LLVM Headers at : ${MLIR_LLVM_INCLUDEPATH}")
+
+    target_include_directories(mlir_backend PRIVATE ${MLIR_INCLUDE_PATHS}) 
+    
+    llvm_map_components_to_libnames(llvm_libs support core irreader)
+    
+    # Link MLIR libs
+    target_link_libraries(
+      mlir_backend PRIVATE 
```

Comment:
Indent is 4 for cmake files. fix through the whole file.

Reply:
Fixed.